### PR TITLE
fix traffic sol

### LINKED ISSUE #4072: Measuring Traffic Wrong Sol
https://forum.usaco.guide/t/usaco-2019-february-contest-bronze-problem-3-measuring-traffic/5183/1

Both the model solution and the USACO Guide solution are wrong. Though they agree on the test data.

### PR DESCRIPTION
**Pull request by SansPapyrus683**
_Place an "x" in the corresponding checkbox if it is done or does not apply to this pull request._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions).
  - I understand that if it is clear that I have not attempted to follow these conventions, my PR will be closed.
  - If changes are requested, I will re-request a review after addressing them.
- [x] I have linked this PR to any issues that it closes.

closes #4072

### CODE REVIEW

**Comment by SansPapyrus683** [comment]
@bqi343 do you suppose i should add a note saying "btw official sol is wrong lol"

**Comment by bqi343** [comment]
ok

File: solutions/bronze/usaco-917.mdx [hunk 1]
```diff
@@ -7,7 +7,37 @@ author: Brad Ma, Kevin Sheng, Jay Fu
...
+```
+
+However, the actual answer is
+
+```
+0 5
+5 10
+```
+
+This is because the starting range in the solution was initialized as ($-\infty$, $\infty$) when in reality it should be (0, $\infty$).
```
Comment (bqi343) [inline solutions/bronze/usaco-917.mdx:35]: use `[]` instead of `()` for ranges, since they're inclusive

**Review by bqi343 — CHANGES_REQUESTED** [review]

File: solutions/bronze/usaco-917.mdx (hunk 1 above)
Comment (bqi343) [reply solutions/bronze/usaco-917.mdx:35]: same below

Comment (SansPapyrus683) [reply solutions/bronze/usaco-917.mdx]: should be fixed

**Review by bqi343 — APPROVED** [review]

**Review by ryanchou-dev — APPROVED** [review]
🥶🥶

**Pull request merged**